Models/Courses.cs

```diff
@@ -0,0 +1,29 @@
+﻿
+using System.ComponentModel.DataAnnotations;
+using Microsoft.AspNetCore.Identity;
+namespace UpSkillz.Models;
+
+public class Courses
+{
+    [Key]
+    public int Id { get; set; }
```

Comment:
It's better to call it CourseId, which will be helpful when making many-to-many declarations and will not cause confusion.